releasing.rst

```diff
@@ -21,100 +17,119 @@ we do a few things to prepare:
+4. Review all of the outputs (e.g., plots)
+   generated from the test suite for abnormalities.
 
-5. Run all tests with the Nengo OCL backend.
+5. Run all tests for projects that depend on your project.
+   For example, Nengo OCL depends on Nengo core
+   so we run those tests with
 
    .. code:: bash
 
-      py.test --pyargs nengo --plots --simulator nengo_ocl.Simulator
+      pytest --pyargs nengo --plots --simulator nengo_ocl.Simulator
 
-   If any tests fail, attempt to fix them by changing Nengo.
-   If they cannot be fixed in Nengo, then
-   `file an issue <https://github.com/nengo/nengo_ocl/issues>`_.
+   If any tests fail, attempt to fix them in your project.
+   If they cannot be fixed and require upstream changes,
+   then file an issue with that project or contact its maintainer
+   to determine how to proceed.
 
-6. Build the documentation and review all of the rendered
-   examples for abnormalities or unclear figures.
+6. Build documentation and review it for abnormalities.
 
 7. Commit all changes from the above steps before moving on to the next stage.
 
-.. todo::
-
-   Currently we do not run slow tests on all platforms (Windows, Mac OS X, Linux
-   with 32-bit and 64-bit versions of Python 2.7, 3.3, 3.4, and 3.5).
-   Doing so is difficult without dedicated hardware.
-
-Note that any possibly controversial fixes done as a result of
+Note that any fixes done as a result of
 Stage 1 should be done through the normal process of making
 a pull request and going through review.
 However, from Stage 2 onward, the work is done directly
 on the ``master`` branch.
-It can therefore result in bad things,
-so proceed with caution!
+Proceed with caution!
 
 Stage 2: Releasing
 ==================
 
 Once everything is prepared, we're ready to do the release.
 It should be okay to work in the same directory that you
 do development, but if you want to be extra safe,
-you can do a fresh clone of Nengo into a separate directory.
+you can do a fresh clone of the repository
+into a separate directory.
 
-1. Change the version information in ``nengo/version.py``.
+1. Change the version information for your project.
+   For most Python projects, it lives in ``project/version.py``.
    See that file for details.
 
-2. Set the release date in the changelog (``CHANGES.rst``).
+2. Set the release date in the changelog (usually ``CHANGES.rst``).
+
+3. Run ``collective.checkdocs`` to ensure proper formatting for PyPI.
+
+   .. code:: bash
+
+      python setup.py checkdocs
 
-3. ``git add`` the changes above and make a release commit with
+4. ``git add`` the changes above and make a release commit with
 
    .. code:: bash
 
-      git commit -m "Release version $(python -c 'import nengo; print(nengo.__version__)')"
+      git commit -m "Release version X.Y.Z"
 
-4. Review ``git log`` to ensure that the version number is correct; if not,
+5. Review ``git log`` to ensure that the version number is correct; if not,
```

Comment:
Since you removed the Python command in the previous step, would a person still be able to see the mistake using `git log`?